crates/viewer/re_blueprint_tree/src/blueprint_tree.rs

```diff
@@ -157,6 +157,10 @@ impl BlueprintTree {
                                 root_container,
                             );
                         }
+                    })
+                    .response
+                    .widget_info(|| {
+                        WidgetInfo::labeled(WidgetType::Panel, true, "_blueprint_tree")
```

Comment:
If someone has a screen reader enabled on web, I think it will read this on focus (although would be good to double check). So maybe this should be a more human friendly label like "Blueprint Tree"? I guess you want to be sure that it's unique though...

Otherwise we could maybe only add these labels when the testing feature is enabled?

Reply:
I think it definitely can be enabled outside egui, but in this case it wouldn't be much of a help to anyone trying to visualize data...

Labels starting with underscore are not visible on the screen. Every other label should be visible to the user.